### ISSUE TITLE
CreateBitmap Function in "gdi32" Library

### ISSUE THREAD

**Issue by GCuser99** [fafalone/WinDevLib#21]
Jon, I could not find this one in your library:
```vba
Public Declare PtrSafe Function CreateBitmap Lib "gdi32" (ByVal nWidth As Long, ByVal nHeight As Long, ByVal nPlanes As Long, ByVal nBitCount As Long, lpBits As Any) As LongPtr
```

Thx!

**Comment by fafalone** [comment]
Wow I don't know how many times I've been over wingdi.h; must have just assumed it was there lol. Will definitely be in the next update.

**Comment by GCuser99** [comment]
No problem and no hurry. I'm sure you'll get more as people use it. Thx - WDL is awesome!

**Comment by fafalone** [comment]
This API is now available in v7.10.396, thanks for letting me know!
